### ISSUE TITLE
In v2.1.0, conversion of "tensorflow.python.framework.ops.Tensor" to array 

### ISSUE THREAD

**Issue by shivamdubey7466** [tensorflow/tensorflow#38415]
I have used **tf.nn.softmax_cross_entropy_with_logits(logits,labels)** , which as an output gives a tensor of type "tensorflow.python.framework.ops.Tensor". 
I need to convert it into numpy array which by any means i am not able to do.
I have even tried to create tf.Session() but that to not working in this case.
Please help me out in it.

**Comment by gadagashwini** [comment]
@shivamdubey7466, Can you provide the complete code to reproduce the issue. Thanks!

**Comment by shivamdubey7466** [comment]
Hi, I hadn't initialized the global_variable_initializer() . The Mistake was from my end. Thanks for your support @gadagashwini
